src/app/services/tasksService/tasks.service.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { TestBed } from '@angular/core/testing';
+import { TasksService } from './tasks.service';
+
+describe('TasksService', () => {
+  let tasksService;
+  beforeEach(() => {
+    tasksService = TestBed.get(TasksService);
+  });
+
+  it('should be created', () => {
+    expect(tasksService).toBeTruthy();
+  });
+
+  it('getAllTasks() should return data', done => {
+    tasksService.getAllTasks().subscribe(res => {
+      expect(typeof res).toEqual('object');
+      done();
+    });
+  });
+
+  it('getAllTasks() should return 6 objects', done => {
+    tasksService.getAllTasks().subscribe(res => {
+      expect(res.length).toEqual(6);
+      done();
+    });
+  });
+
+  it('should return tableDataItem', () => {
```

Comment:
Please add `done` function and invoke it inside subscription as it is implemented in other tests.